compiler/record-minmax/include/RecordFunction.h

```diff
@@ -33,12 +33,6 @@ float getNthPercentile(std::vector<float> &vector, float percentile)
   if (percentile < 0 || percentile > 100)
     throw std::runtime_error("Percentile must be ranged from 0 to 100");
 
-  if (percentile == 0.0)
-    return vector.front();
```

Comment:
The problem here: vector is not sorted at this point.

Reply:
Wow..